apps/ledger-live-desktop/src/renderer/modals/Send/steps/StepSummary.tsx

```diff
@@ -84,8 +84,13 @@ const StepSummary = (props: StepProps) => {
   const specific = currency ? getLLDCoinFamily(mainAccount.currency.family) : null;
   const SpecificSummaryNetworkFeesRow = specific?.StepSummaryNetworkFeesRow;
 
-  const memo = "memo" in transaction ? transaction.memo : undefined;
-
+  const memo = lldMemoTag?.enabled
```

Comment:
Can we please create a dedicated function to retrieve the memo/tag?
Can we use something like transaction.family to know which coin we are creating a tx for, please?